packages/fx-core/resource/package.nls.json

```diff
@@ -194,5 +194,72 @@
   "core.getUserEmailQuestion.title": "Add owner to Teams/AAD app for the account under the same M365 tenant (email)",
   "core.getUserEmailQuestion.validation1": "Email address cannot be null or empty",
   "core.getUserEmailQuestion.validation2": "Please change [UserName] to the real user name",
-  "core.getUserEmailQuestion.validation3": "Email address is not valid"
+  "core.getUserEmailQuestion.validation3": "Email address is not valid",
+  "core.boot.DetailedError": "Detailed error",
+  "core.boot.DetailedErrorReason": "Reason",
+  "core.boot.UnhandledError": "Got an unhandled error",
+  "core.boot.ErrorSuggestions": "Suggestions",
+  "core.boot.InvalidValue": "is invalid with value",
+  "core.boot.SelectValidValues": "Please select valid values for",
+  "core.boot.SomethingIsMissing": "is missing.",
+  "core.boot.SomethingNotFound": "is not found.",
+  "core.boot.SomethingNotExisting": "is not existing.",
+  "core.boot.FailedToCreate": "Failed to create",
+  "core.boot.FailedToProvision": "Failed to provision",
+  "core.boot.FailedToUpdateConfigs": "Failed to update configs for",
+  "core.boot.FailedListPublishingCredentials.": "Failed to list publishing credentials.",
+  "core.boot.FailedDeployZipFile": "Failed to deploy zip file.",
+  "core.boot.FailedUpdateMessageEndpoint": "Failed to update message endpoint with",
+  "core.boot.DownloadFail": "Failed to download from",
+  "core.boot.ClickGetHelp": "Please click 'Get Help' button for more details.",
+  "core.boot.FailToRun": "Failed to run",
+  "core.boot.PerformCommand": "Perform command",
+  "core.boot.Before": "before",
+  "core.boot.FailToCallAppStudioApi": "Failed to call App Studio's api to check AAD application's existence.",
+  "core.boot.SuccessfullyRetrievedZip": "Successfully retrieved zip package from",
+  "core.boot.FallingUseLocalTemplate": "Falling back to use local template zip.",
+  "core.boot.ResourceProvider": "Resource Provider",
+  "core.boot.AlreadyExists": "already exists, do not need to register.",
+  "core.boot.BotResourceExists": "Bot resource already existed on",
+  "core.boot.SkipCreateBotResource": "skip creating Bot resource.",
+  "core.boot.WorkingDirMissing": "Working directory is missing.",
+  "core.boot.FailRetrieveAzureCredentials": "Failed to retrieve Azure credentials.",
+  "core.boot.LoginToAzure": "Login to Azure.",
+  "core.boot.SkipDeployment": "Skipping deployment: no updates found.",
+  "core.boot.ScaffoldingBot": "Scaffolding bot.",
+  "core.boot.ScaffoldingBotSuccess": "Successfully scaffolded bot.",
+  "core.boot.PreProvisionBot": "Pre-provisioning bot.",
+  "core.boot.ProvisionBot": "Provisioning bot.",
+  "core.boot.PreDeployingBot": "Pre-deploying bot.",
+  "core.boot.DeployingBot": "Deploying bot.",
+  "core.boot.DeployingBotSuccess": "Successfully deployed bot.",
+  "core.boot.GeneratingBotARMTemplates": "Generating ARM templates of bot.",
+  "core.boot.GeneratingBotArmTemplatesSuccess": "Successfully generating ARM templates of bot.",
```

Comment:
Using `ARM` or `Arm` should be consistent here.

Reply:
fixed